static/js/dashboard.js

```diff
@@ -1178,8 +1185,13 @@ $(document).ready(function () {
       url: "/dashboard/data/proposals" + qryStr,
       success: function (result) {
         var t1 = Date.now()
+        var chart = $("#proposed-chart").highcharts()
```

Comment:
**Minor:** You don't need this here, it would be better placed between lines 1191 and 1192, would it not?
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/709974d262b28421debbd08243ad3a03ca6f575f/static/js/dashboard.js#L1191-L1192